gensim/sklearn_api/atmodel.py

```diff
@@ -18,17 +37,79 @@
 
 
 class AuthorTopicTransformer(TransformerMixin, BaseEstimator):
-    """
-    Base AuthorTopic module
-    """
+    """Base Author Topic module, wraps :class:`~gensim.models.atmodel.AuthorTopicModel`.
 
+    For more information on the inner workings please take a look at the original class. The model's internal workings
+    are heavily based on `"The Author-Topic Model for Authors and Documents", Osen-Zvi et. al 2004
+    <https://mimno.infosci.cornell.edu/info6150/readings/398.pdf>`_.
```

Comment:
Maybe mention paper only in original class (not here), wdyt?

Reply:
I will replace references with `some text <http://...>`_